package/jest.config.js

```diff
@@ -5,4 +5,5 @@ module.exports = {
   // Ignore lib folder - contains build artifacts and should
   // not be probed for tests
   modulePathIgnorePatterns: ["<rootDir>/lib/typescript"],
+  testRegex: ".spec.ts$",
```

Comment:
Is this necessary? It already detects .spec.ts / .spec.tsx files as default.

Reply:
yes for setup.ts to not be treated as a test

Reply:
I don't think this is better unless we use a different pattern (like a folder to put files like this).